code/src/layouts/app-banner/messages/en.json

```diff
@@ -1,4 +1,12 @@
 {
   "appBanner.header": "Incredible Prices on All Your Favorite Items",
-  "appBanner.paragraphfirst": "Wellcome to our store! Here you will find the best gadgets from around the world, and you can choose the perfect product for you."
+  "appBanner.paragraphfirst": "Wellcome to our store! Here you will find the best gadgets from around the world, and you can choose the perfect product for you.",
```

Comment:
also `typo` here for `Wellcome`
and check this and other properties it should be `camelCase` in your example `appBanner.paragraphFirst`
should to check your other properties in your case

Reply:
I haven't worked on app Banner component, but okay will fix it.